src/components/transactions/Emode/EmodeNaming.ts

```diff
@@ -1,13 +1,6 @@
-export const getEmodeMessage = (categoryId: number, baseAssetSymbol: string): string => {
-  if (categoryId === 0) {
+export const getEmodeMessage = (label: string): string => {
```

Comment:
it's probably a bit stupid to keep the function, but was the "least effect" change